ArtOfIllusion/src/artofillusion/util/FileUtils.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright (C) 2017 by Maksim Khramov
+
+   This program is free software; you can redistribute it and/or modify it under the
+   terms of the GNU General Public License as published by the Free Software
+   Foundation; either version 2 of the License, or (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT ANY 
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A 
+   PARTICULAR PURPOSE.  See the GNU General Public License for more details. */
+
+package artofillusion.util;
+
+import java.io.File;
+
+/**
+ *
+ * @author MaksK
+ */
+public final class FileUtils
```

Comment:
Do you have plans for other file utils?

I'd rather not create a new class for a single method. I also think that unrelated refactors, such as the changes to `POVExporter` do __not__ belong in this PR.